src/components/Topics.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { Link } from "@mui/material";
+import fetchTopics from "../controllers/fetchTopics";
+import { Link as RouterLink } from "react-router-dom";
+import { Paper, Typography, Tooltip } from "@mui/material";
```

Comment:
You appear to have two imports for '@mui/material - is it possible to neaten this up?